### DISCUSSION TITLE
Support `.python-version` with multiple versions for `pyenv` manager

### DISCUSSION THREAD

**Discussion by mkniewallner** [renovatebot/renovate#31837] [Suggest an Idea]
### Tell us more.

Renovate [implements `pyenv` manager](https://github.com/renovatebot/renovate/tree/main/lib/modules/manager/pyenv) to update Python version defined in a `.python-version` file. Per [`pyenv` documentation](https://github.com/pyenv/pyenv/blob/d201daeb3366cae98cca88654b4c52018175ded5/README.md?plain=1#L155-L164), it is also possible to set multiple Python versions in a single `.python-version` file, separated by newlines:
> You can also specify multiple versions in a `.python-version` file by hand,
separated by newlines. Lines starting with a `#` are ignored.

Would it make sense for Renovate to support that, by extracting all possible Python versions defined in the file? Currently, [the content of the file is trimmed](https://github.com/renovatebot/renovate/blob/03a85316b20c8eb21dad2e8a0d14cfe2b739e43d/lib/modules/manager/pyenv/extract.ts#L8), assuming that only one version can be specified.

If it makes sense, I'm willing to implement support for it.

**Comment by rarkins** [comment]
Sure, I've converted it to an issue here: https://github.com/renovatebot/renovate/issues/31850
